### DISCUSSION TITLE
global_table 2分10s延迟删除太慢了，在高并发下，这张表数据太多了，有啥优化方案吗

### DISCUSSION THREAD

**Discussion by PeppaO** [apache/incubator-seata#6616] [Q&A]
global_table 2分10s延迟删除太慢了，在高并发下，这张表数据太多了，有啥优化方案吗？
除了调低这个130s的参数，还有其他方案吗？
如果等它低频的时候删除完毕，我很担心在高频的时候随着global_table的堆积，影响tps，浪费数据库磁盘空间
@slievrly @funky-eyes @lightClouds917 @wangliang181230

**Comment by slievrly** [comment]
1. Increase queryLimit
2. Asynchronous task to clean up global_table and perform data sharding
